### PR TITLE
Install scripts & separate out Grafana

### PR DESCRIPTION
**Pull request by coverbeck**
SEAB-1471

First step towards using AWS CLI to install CloudFormation stacks.

##Advantages

* Lets installer tweak settings locally in a file instead of
clunkily through the UI.
* Can automatically handle stuff that CloudFormation can't, e.g., 
update Route53.

##Grafana

The AWS CLI has a limit on the size of the template file. Our Dockstore
template was too large. So split out the Grafana into its own file. Making
excuses, but this may be for the best anyway:

1. That large of a template file becomes unreadable (they still sort of are).
2. We may eventually want to separate Grafana deploys from Dockstore;
Grafana changes less frequently than Dockstore, but we've been pushing
out a new version every time Dockstore changes.

### CODE REVIEW

**Comment by denis-yuen** [comment]
> So split out the Grafana into its own file. Making
> excuses, but this may be for the best anyway:

Having smaller templates might make it easier to convert to the CDK some-day anyway https://docs.aws.amazon.com/cdk/latest/guide/home.html#developing

File: scripts/templates/grafana-deploy.prod.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+STACK_NAME="Grafana-0731"
```
Comment (garyluu) [inline scripts/templates/grafana-deploy.prod.properties:1]: Are they supposed to have the same STACK_NAME?

Comment (coverbeck) [reply scripts/templates/grafana-deploy.prod.properties]: Good catch, I meant to remove these files entirely, and thought I had. At one point I had separate scripts/settings for Dockstore/Grafana, but ended up combining them.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
 # Don't accidentally start a job on CircleCI with a private repo
 .circleci/
 generated/
+scripts/dockstore-deploy.properties
+scripts/grafana-deploy.properties
```
Comment (garyluu) [inline .gitignore:5]: Assuming this is still needed

Comment (coverbeck) [reply .gitignore]: Sigh, it wouldn't hurt, but no, it shouldn't be there.

**Pull request merged**